docs/gitpod/use_gitpod.md

```diff
@@ -26,4 +28,4 @@
 
     ![](devtools.png)
 
-## Note: it's also ok to initialize Cyberbrain first, then run the program.
\ No newline at end of file
+## Note: it's also ok to initialize Cyberbrain first, then run the program.
```

Comment:
Thanks the PR! May I ask what's changed here?

Reply:
Looks like vim add a `NewLine` char for me.
![image](https://user-images.githubusercontent.com/9675939/95148339-9f6b0c80-07b5-11eb-9875-d8d2e89473e7.png)

which is `0x0a`

![image](https://user-images.githubusercontent.com/9675939/95148362-ab56ce80-07b5-11eb-9d9e-4ee9db376082.png)